mp/src/game/server/momentum/mom_player.h

```diff
@@ -225,9 +225,9 @@ class CMomentumPlayer : public CBasePlayer, public CGameEventListener, public CM
     void SetCurrentZoneTrigger(CTriggerZone *pZone) { return m_CurrentZoneTrigger.Set(pZone); }
     CTriggerZone *GetCurrentZoneTrigger() const { return m_CurrentZoneTrigger.Get(); }
 
-    void CreateStartMark();
+    bool CreateStartMark();
     SavedLocation_t *GetStartMark(int track) const { return (track >= 0 && track < MAX_TRACKS) ? m_pStartZoneMarks[track] : nullptr; }
-    void ClearStartMark(int track);
+    bool ClearStartMark(int track, bool bSuppressPrintout = false);
```

Comment:
It's better practice to make booleans naturally true, so that way the brain isn't doing double negatives when parsing logic. "Should *not* *not print message*" versus "should print message"

Reply:
Never given that much thought but it makes a lot of sense. I'll stop doing it :)